scripts/build_binary_and_layer_dockerized.sh

```diff
@@ -17,6 +17,15 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
+if [ -z "$AGENT_VERSION" ]; then
+    echo "No agent version set, agentVersionDefault will be used instead (for the metrics log intake endpoint)."
+    read -p "This should only be done when building for staging or testing. Continue (y/n)?" CONT
```

Comment:
Instead of printing `"This should only be done when building for staging or testing. Continue (y/n)?"` Can we make it so that if you are publishing for prod, the agent version is required, but if you are publishing for staging, the default version is automatically used? That way, we can replace a prompt with hardcoded logic.

Reply:
I think we have separate files for releasing to prod and sandbox, maybe the logic can go in those files

Reply:
Removed this warning. The only script where the agent version is required is `publish_prod.sh`. For `publish_sandbox.sh` as well as the local testing `build_docker_python.sh`, an agent version can still be specified otherwise the default is used.